### DISCUSSION TITLE
Make it responsive

### DISCUSSION THREAD

**Discussion by IndigoBee** [plausible/analytics#319] [Anything Else]
It currently looks very bad on mobile. Do you plan to improve the responsiveness soon?

**Comment by ukutaht** [comment]
Sorry I'm not a designer, doing my best at the moment and I'm the first to admit that mobile UX leaves a lot to be desired.

We will improve it in the future for sure.

**Comment by IndigoBee** [comment]
Here's what I mean by that:

1. You can't see the last column, and when the text is long, number and text overlap
![photo5992534505811981550](https://user-images.githubusercontent.com/33088396/93591091-e8f7e100-f9d9-11ea-88d7-782779290108.jpg)

2. The graph is so small..
![photo5992534505811981549](https://user-images.githubusercontent.com/33088396/93591093-ea290e00-f9d9-11ea-891a-c94a42555aa5.jpg)
